File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/proposal/GraduationDesignDatumGroupBean.java
package top.zbeboy.isy.web.bean.graduate.design.proposal;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.GraduationDesignDatumGroup;

@EqualsAndHashCode(callSuper = false)
@Data
public class GraduationDesignDatumGroupBean extends GraduationDesignDatumGroup {
    private String size;
    private String originalFileName;
    private String newName;
    private String relativePath;
    private String ext;
    private String uploadTimeStr;
    private String graduationDesignReleaseId;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/proposal/GraduationDesignDatumGroupBean.kt
package top.zbeboy.isy.web.bean.graduate.design.proposal

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignDatumGroup

/**
 * Created by zbeboy 2018-01-29 .
 **/
class GraduationDesignDatumGroupBean : GraduationDesignDatumGroup() {
    var size: String? = null
    var originalFileName: String? = null
    var newName: String? = null
    var relativePath: String? = null
    var ext: String? = null
    var uploadTimeStr: String? = null
    var graduationDesignReleaseId: String? = null
}